ISSUETEMPLATE.md

```diff
@@ -0,0 +1,30 @@
+## LGT GridIron Project
+
+## Issue Title:
```

Comment:
Need to add in for the author to check if they used conventional commits and included the github ticket # into the commit. If not, they need to manually link the two.